.drone.star

```diff
@@ -0,0 +1,100 @@
+OC_CI_PHP = "owncloudci/php:%s"
+OC_UBUNTU = "owncloud/ubuntu:20.04"
+PLUGINS_GITHUB_RELEASE = "plugins/github-release"
+
+DEFAULT_PHP_VERSION = "8.1"
+
+dir = {
+    "base": "/var/www/owncloud",
```

Comment:
```suggestion
    "base": "/var/www/owncloud",
```
IMO this is ocis sdk not owncloud so `/var/www/owncloud` could be something more suitable

Reply:
we can use the default base (`./drone/src`)